after_compile/amazonlinux2_lambda.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# We need only binary wkhtmltox files in lambda layer, so we delete all the others
+rm -rf /tgt/wkhtmltox/{include,share,lib}
+
+# We need to copy dependencies which are not presented in amazonlinux2 image
+libs="libjpeg libpng15 libXrender libfontconfig libfreetype libXext libX11 libxcb libXau libexpat libuuid libbz2"
```

Comment:
Wonder if it's possible to parse the outputs of `ldd` instead of hardcoding this.

Reply:
Sounds like a promising idea. Is it for future improvements, or it makes sense to spend time on investigation now?

Reply:
Let's leave it for now :+1: